specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -2584,8 +2584,9 @@
       "type": "object",
       "properties": {
         "reportUrl": {
-          "description": "The URL to download the generated report.",
-          "type": "string"
+          "description": "The CSV file from the reportUrl blob link is consists of reservation usage data with the following schema at daily granularity",
```

Comment:
nit: remove extra is - The CSV file from the reportUrl blob link consists of reservation usage data with the following schema at daily granularity